lib/mapObjects/CommonConstructors.h

```diff
@@ -33,19 +33,11 @@ class CDefaultObjectTypeHandler : public AObjectTypeHandler
 		auto obj = new ObjectType();
 		preInitObject(obj);
 
+		//Set custom template or leave null
 		if (tmpl)
 		{
 			obj->appearance = tmpl;
 		}
```

Comment:
```suggestion
		obj->appearance = tmpl;
```

Reply:
Actually it would change logic. Now if tmpl is nullptr (default), the original template, if existing, is not modified.